locale/fr_FR/emails.po

```diff
@@ -1374,12 +1374,12 @@ msgstr ""
 "\n"
 "{$name}, <br />\n"
 "<br />\n"
-"Le rapport d'activité éditoriale de votre revue pour {$month}, {$year} est "
+"Le rapport d'activité éditoriale de votre revue pour {$month} {$year} est "
 "maintenant disponible. Les principales statistiques pour ce mois sont "
 "affichées ci-dessous.<br />\n"
 "<ul>\n"
 "\t<li> Nouvelles soumissions pour ce mois : {$newSubmissions} </li>\n"
-"\t<li> Soumissions refusées pour ce mois : {$rebuildsubmissions} </li>\n"
+"\t<li> Soumissions refusées pour ce mois : {$declinedsubmissions} </li>\n"
```

Comment:
I think this should be `declinedSubmissions` (note capitalization).

Reply:
You're totally right